app/models/council.rb

```diff
@@ -2,4 +2,8 @@ class Council < ActiveRecord::Base
   belongs_to :user
   has_many :council_memberships
   has_many :members, through: :council_memberships
+  has_many :questions
+
+  validates :council_name, uniqueness: true, presence: true
+  validates :user_id, presence: true
```

Comment:
It may be better to validate user rather than user_id. validates :user would check that the user_id column has a value, and that it points to a valid user record. This is a better check than just validates user_id. The later would accept a user_id that doesn't identify a user record. It's just a not null check.